src/app/components/layout/search-bar/search-bar.component.ts

```diff
@@ -1,9 +1,11 @@
 import { Component, ElementRef, ViewChild, OnDestroy } from '@angular/core';
 import { Router } from '@angular/router';
 import { TranslateService } from '@ngx-translate/core';
-import { Subscription } from 'rxjs';
+import { Subscription, of } from 'rxjs';
+import { delay, mergeMap } from 'rxjs/operators';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
import { delay, mergeMap } from "rxjs/operators";
```